pom.xml

```diff
@@ -73,6 +73,15 @@
         <vendor.name>Eclipse Project for JAX-WS</vendor.name>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
```

Comment:
consider adding the dep definition to dependencyManagement. Probably not a big deal if it stays...

Reply:
This is single pom project so it will just add more lines to the pom. But I'll do it.